src/components/send/UploadQRModal.tsx

```diff
@@ -138,9 +138,9 @@ export const UploadQRModal = ({
                             maxSize={5}
                             onSizeError={handleSizeError}
                             onTypeError={handleTypeError}
-                            tabIndex={-1}
+                            classes='focus-within:!outline-theme-primary-600 rounded-2xl'
```

Comment:
I am using `focus-within` and forcing the class name due to the library not allowing the management of tab index or overriding their styles.